packages/cursorless-vscode/src/ScopeSupportTreeProvider.ts

```diff
@@ -0,0 +1,198 @@
+import { CursorlessCommandId } from "@cursorless/common";
+import {
+  ScopeProvider,
+  ScopeSupport,
+  ScopeSupportLevels,
+  ScopeTypeInfo,
+} from "@cursorless/cursorless-engine";
+import * as vscode from "vscode";
+import { VisualizationType } from "./ScopeVisualizerCommandApi";
+
+export class ScopeSupportTreeProvider
+  implements vscode.TreeDataProvider<MyTreeItem>
+{
+  private onDidChangeScopeSupportDisposable: vscode.Disposable | undefined;
+  private treeView: vscode.TreeView<MyTreeItem>;
+  private supportLevels: ScopeSupportLevels = [];
+
+  private _onDidChangeTreeData: vscode.EventEmitter<
+    MyTreeItem | undefined | null | void
+  > = new vscode.EventEmitter<MyTreeItem | undefined | null | void>();
+  readonly onDidChangeTreeData: vscode.Event<
+    MyTreeItem | undefined | null | void
+  > = this._onDidChangeTreeData.event;
+
+  constructor(
+    private context: vscode.ExtensionContext,
+    private scopeProvider: ScopeProvider,
+  ) {
+    this.treeView = vscode.window.createTreeView("cursorlessScopeSupport", {
```

Comment:
Since we get a command for free, should we use dots in this so all the commands Cursorless adds are in the same namespace?
```suggestion
    this.treeView = vscode.window.createTreeView("cursorless.scopeSupport", {
```

Reply:
huh yeah that makes sense to me